vie.ini

```
@@ -3,10 +3,442 @@
 ; A message doesn't need to be there to be shown correctly on the client.
 ; Only messages shown in PocketMine itself need to be here
 
+language.name=Tiếng Việt
+language.selected=Đã chọn {%0} ({%1}) là ngôn ngữ cơ bản
 
+ability.flight=Flying
```

"Đang bay" or "Bay"?
```suggestion
ability.flight=Đang bay
```

---

`Flying` is a noun so we should use the word `Bay`